packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1757,6 +1758,18 @@ export default class Meeting extends StatelessWebexPlugin {
     return this.members.addMember(invitee, alertIfActive);
   }
 
+  /**
+   * cancel a PSTN call made during meeting
+   * @param {Object} invitee
+   * @param {String} invitee.phoneNumber
+   * @returns {Promise} see #members.cancelInvite
+   * @public
+   * @memberof Meeting
+   */
+  cancelInvite(invitee) {
```

Comment:
can we change the comment , And this is specific to phone number can we can endPhoneNumber of something cancelInvite will confuse user as they will think its for every member

Reply:
@arun3528 sure, yes this specific to phone so when a user calls and wants to hang up before caller answers the phone we call this method but once they are in the meeting we call remove, can we call cancelPhoneInvite?